src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.scss

```diff
@@ -1,6 +1,21 @@
 .schema-info-viewer {
     overflow: auto;
 
+    &__row {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        align-items: flex-start;
+        gap: 50px;
```

Comment:
Flex gap is not supported id 14 Safari

Reply:
And still it is used all over the project. We probably should address that, if we target safari 14.
Are you suggesting we don't use it here?